pkg/app/server/config/config.go

```diff
@@ -162,15 +177,17 @@ func parseFlags() *Config {
   status - status of the service`)
 	flag.StringVar(&c.LogDir, "logdir", "", "System option to define log directory to vxserver")
 	flag.StringVar(&c.OtelAddr, "oteladdr", "", "System option to define log opentelemetry address")
+	flag.IntVar(&c.MaxConcSyncingAgents, "mcsa", 0, "Maximum agents to synchronise concurrently")
```

Comment:
Would this "0" default value replace default value from defaultConfig? Seems like in case if env var is not set and we just start with default settings MaxConcSyncingAgents would be set to 0. Maybe it is better to do something like:
```suggestion
	flag.IntVar(&c.MaxConcSyncingAgents, "mcsa", defaultConfig.MaxConcSyncingAgents, "Maximum agents to synchronise concurrently")
```